ci/run_ci.sh

```diff
@@ -96,13 +96,15 @@ fi
 # HERA role.jedipara can not use /scratch1/NCEPDEV/global. MSU role-da
 # can not use /work2/noaa/global. The logic below modifies the paths so
 # role.jedipara and role-da can run g-w based ctests.
-if [[ "${TARGET}" = "hera" ]]; then
+if [[ $TEST_WORKFLOW == 1 ]]; then
```

Comment:
seems strange to set this equal to 1, but I guess that is outside of the control of GDASApp/this PR?

Reply:
`TEST_WORKFLOW` are local to `ci/run_ci.sh` and `ci/driver.sh`.  Both scripts were already using `TEST_WORKFLOW=1` trigger the running of g-w based GDASApp ctests.  I simply replicated existing logic here.  That said, one could replace the `0/1` toggle with `NO/YES`.

Reply:
No, this is fine, for all I know I was the one who did that in the first place (haha)